test/webhook_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sed -ri "s|'(quay.io/reactiveops/polaris:).+'|'\1${CIRCLE_SHA1}'|" ./deploy/webhook.yaml
```

Comment:
I'm usually guilty of not doing this enough, but we should always try to comment regex for the intended results (for future reference of intent).